cmd/wego/version/cmd.go

```diff
@@ -22,16 +22,27 @@ var Cmd = &cobra.Command{
 }
 
 func runCmd(cmd *cobra.Command, args []string) {
-	if checkResponse, err := checkpoint.Check(&checkpoint.CheckParams{
+	fmt.Println("Current Version:", Version)
+	fmt.Println("GitCommit:", GitCommit)
+	fmt.Println("BuildTime:", BuildTime)
+	fmt.Println("Branch:", Branch)
+	fmt.Println("Flux Version:", version.FluxVersion)
+}
+
+func CheckVersion() (string, error) {
+	checkResponse, err := checkpoint.Check(&checkpoint.CheckParams{
 		Product: "weave-gitops",
 		Version: Version,
-	}); err == nil && checkResponse.Outdated {
-		log.Infof("wego version %s is available; please update at %s",
-			checkResponse.CurrentVersion, checkResponse.CurrentDownloadURL)
-	} else {
-		fmt.Println("Version", Version)
-		fmt.Println("GitCommit:", GitCommit)
-		fmt.Println("BuildTime:", BuildTime)
-		fmt.Println("Branch:", Branch)
+	})
+
+	if err != nil {
+		return "", fmt.Errorf("Unable to retrieve latest version: %v", err)
 	}
+
+	if err == nil && checkResponse.Outdated {
```

Comment:
Not need to check `err==nil` here as it is inferred for the previous `if` statement.

Reply:
Whoops forgot about that after changing things up. Good catch.